recipe/meta.yaml

```diff
@@ -1,37 +1,61 @@
-{% set version = "0.7.0" %}
-{% set sha256 = "1aaadcd69338252ade4f7ec1265e1a19184bf916d84c9b7df095f423948cb89f" %}
+{% set version = "0.10.0" %}
 
 package:
   name: build
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/b/build/build-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/pypa/build
+  git_rev: {{ version }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - pyproject-build = build.__main__:entrypoint
 
 requirements:
+  build:
+    - git  # [not win]
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
```

Comment:
```suggestion
```
Wheel is not needed. `build` uses `flit-core`, which creates wheels without using the `wheel` package. Ignore the linter warning. The linter will be fixed in https://github.com/anaconda-distribution/anaconda-linter/pull/272.

Reply:
OK!